src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/ComponentIdentifier.cs

```diff
@@ -53,7 +53,7 @@ private static ProjectComponents GetSDKProjectComponents(IProject project, IProj
                 components |= ProjectComponents.WindowsDesktop;
             }
 
-            if (file.Sdk.Equals(MSBuildConstants.DesktopSdk, StringComparison.OrdinalIgnoreCase))
+            if (file.Sdk.Contains(MSBuildConstants.DesktopSdk, StringComparer.OrdinalIgnoreCase))
```

Comment:
These stringcompareres are making it so it's using LINQ rather than you're actual contains method

Reply:
removed